mise.toml

```diff
@@ -47,6 +48,10 @@ binary_signer = "1.0.4"
 forge = "ubi:foundry-rs/foundry[exe=forge]"
 cast = "ubi:foundry-rs/foundry[exe=cast]"
 anvil = "ubi:foundry-rs/foundry[exe=anvil]"
+# aqua backend is currently broken
+shellcheck = "ubi:koalaman/shellcheck"
+# custom asdf plugin for now
+kurtosis = "asdf:sigma/asdf-kurtosis"
```

Comment:
Is it ok if we migrate this to an `ethereum-optimism/asdf-kurtosis` repo?

Reply:
https://github.com/ethereum-optimism/asdf-kurtosis

Reply:
sure thing. I didn't realize there was no real process to create repos. Will move my hack there.
Thanks !

Reply:
https://github.com/ethereum-optimism/asdf-kurtosis/pull/1